src/App/App.js

```diff
@@ -1,23 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import { registerRootComponent } from 'expo';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
 import 'react-native-gesture-handler';
 
-const App = () => {
-  return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-    </View>
-  );
+import Form from '../components/Form/Form.js';
+
+const Stack = createStackNavigator();
+
+class App extends Component {
+  state = {
+      description: ''
+    }
+
+  render() {
+    return (
+      <NavigationContainer>
+        <Stack.Navigator screenOptions={{
+          headerShown: false
+        }}>
+          <Stack.Screen name="Form" component={Form}
+          />
+        </Stack.Navigator>
+      </NavigationContainer>
+    );
```

Comment:
This looks great, I might need to go over it once before I start to implement but looking good!